data/free worlds intro.txt

```diff
@@ -513,7 +513,7 @@ mission "FW Katya 1"
 				`	"Okay, tell me more."`
 					goto `more`
 	
-			`	"Look kid," she says, "when I ask you to pick sides, when I offer you a mission that's gonna burn some bridges for you, I won't be sneaky about it. I'll tell you exactly what you're signing up for. What I'm offering you now is just a chance to get to know us, and let us get to know you, with no expectations. Heck, you can even run missions for the Reps at the same time."`
+			`	"Look, kid," she says, "when I ask you to pick sides, when I offer you a mission that's gonna burn some bridges for you, I won't be sneaky about it. I'll tell you exactly what you're signing up for. What I'm offering you now is just a chance to get to know us, and let us get to know you, with no expectations. Heck, you can even run missions for the Reps at the same time."`
```

Comment:
The original is easily allowable - this is a character's speech, and its punctuation requirements stem from the character and the context, not the formal language rules.

Reply:
I get that characters don't always speak grammatically, but even so my two cents are that it's better with the comma.

Reply:
I think too it is better with the comma, it really shows Katya thinking of what to say to the player.